.env.runtime.schema

```
@@ -38,19 +38,19 @@ LOG_LEVEL=ifs(
 
 # Disables the Changelogs flow
 # @type=boolean
-NUXT_PUBLIC_COMMON_UI_DISABLE_CHANGELOG=false
+NUXT_PUBLIC_COMMON_UI_DISABLE_CHANGELOG=eq($APP_MODE, ci)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)